flexbox/src/main/java/com/google/android/flexbox/FlexboxLayout.java

```diff
@@ -775,6 +785,7 @@ private void layoutVertical(boolean isRtl, boolean fromBottomToTop, int left, in
                 childRight -= mDividerVerticalWidth;
             }
             float spaceBetweenItem = 0f;
+            int visibleCount;
```

Comment:
You can surround each of the switch-case bodies with braces to use the same variable name. That way you can skip preparing this int when it is not used. Not a big deal, though.

Reply:
Done. Thanks.